### PR TITLE
Convert dropdown to radio buttons in tiled gallaries

### LINKED ISSUE #6886: Tiled galleries: Use radio buttons instead of dropdown for types
I've recently done some User Testing on adding and editing galleries that come with the _Tiled Galleries_ from Jetpack. 

In one of the tasks I asked the subject to change the type of the gallery from the default (**Thumbnail Grid**) to **Circles**. I observed how the users usually couldn't find anything related to what the task described, telling me they can't see any "Circles" option. They weren't actually reading the label, just trying to figure it out while browsing around. Afterwards they would navigate back to the editor, add a new gallery and get blocked again, until they decide to click around and actually discover the option was in the dropdown.

My conclusion was that the dropdown is hiding one of the most important information related to the gallery: it's type. So, as a suggestion, I would represent the **Type** option as radio buttons instead of a dropdown menu:

radio_buttons

This may cut some of the back-and-forth navigation I've seen and also encourage faster exploration of the options (also, less clicks 😁  )

It would be nice to hear more about this and the reasoning behind choosing a dropdown for the option and also thoughts on my suggestion :)

### PR DESCRIPTION
**Pull request by shantanu2704**
Fixes #6886

#### Changes proposed in this Pull Request:

* Convert the drop-down menu for _Tiled Galleries_ to _radio_ buttons

### CODE REVIEW

**Comment by dereksmart** [comment]
I'm pretty sure this will need another look, as these settings are filtered into core's `type` setting, which is displayed via a backbone template where these options are hard-coded.  Unfortunately the PR as it stands does not work :(

**Comment by shantanu2704** [comment]
Hi @dereksmart !

Do you mind explaining what you mean by 

> displayed via a backbone template where these options are hard-coded.

Also, what other changes does the PR need?

File: functions.gallery.php [hunk 1]
```diff
@@ -90,7 +90,7 @@ function print_media_templates() {
 				<span><?php _e( 'Type', 'jetpack' ); ?></span>
 				<select class="type" name="type" data-setting="type">
 					<?php foreach ( $this->gallery_types as $value => $caption ) : ?>
-						<option value="<?php echo esc_attr( $value ); ?>" <?php selected( $value, $default_gallery_type ); ?>><?php echo esc_html( $caption ); ?></option>
+						<input type="radio" value="<?php echo esc_attr( $value ); ?>" <?php checked( $value, $default_gallery_type ); ?>><?php echo esc_html( $caption ); ?></input>
```
Comment (oskosk) [inline functions.gallery.php]: An `<input>` should not be a child of a `<select>`

**Comment by simison** [comment]
Now with block editor and all, I think this PR could be closed.

**Comment by jeherve** [comment]
Yes, I think this is fine to close for now.

**Pull request closed without merging**